tools/install_demo_configuration.sh

```diff
@@ -352,7 +352,10 @@ echo "$ADMIN_CERT_KEY" | $SUDO_CMD tee "$ES_CONF_DIR/kirk-key.pem" > /dev/null
 
 echo "" | $SUDO_CMD tee -a  "$ES_CONF_FILE"
 echo "######## Start OpenDistro for Elasticsearch Security Demo Configuration ########" | $SUDO_CMD tee -a "$ES_CONF_FILE" > /dev/null 
-echo "# WARNING: revise all the lines below before you go into production" | $SUDO_CMD tee -a "$ES_CONF_FILE" > /dev/null 
+echo "# WARNING: revise all the lines below before you go into production" | $SUDO_CMD tee -a "$ES_CONF_FILE" > /dev/null
+
+echo "opendistro_security.ssl_dual_mode_enabled: true" | $SUDO_CMD tee -a  "$ES_CONF_FILE" > /dev/null
```

Comment:
need to remove this line?

Reply:
All changes to this file can be reverted